package.json

```diff
@@ -45,12 +46,12 @@
     "@mdx-js/react": "^3.0.0",
     "clsx": "^2.1.1",
     "date-fns": "^3.6.0",
+    "highlightjs-terraform": "https://github.com/highlightjs/highlightjs-terraform#eb1b9661e143a43dff6b58b391128ce5cdad31d4",
```

Comment:
Any reason why this is needed?

Reply:
We used this to enable HCL syntax highlighting in the previous docs site. We're using Docusaurus-native syntax highlighting now, so we shouldn't need this until (potentially) we address #6